src/main/java/pvs/app/service/TrelloApiService.java

```diff
@@ -0,0 +1,108 @@
+package pvs.app.service;
+
+import com.google.gson.Gson;
+import kong.unirest.CookieSpecs;
+import kong.unirest.HttpResponse;
+import kong.unirest.Unirest;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.stereotype.Service;
+import pvs.app.service.data.TrelloData;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Objects;
+
+@Service
+@SuppressWarnings("squid:S1192")
+public class TrelloApiService {
+    private final Runtime rt;
+    private final String trelloApiKey, trelloApiToken;
+
+    public TrelloApiService() {
+        this.trelloApiKey = System.getenv("PVS_TRELLO_KEY");
+        this.trelloApiToken = System.getenv("PVS_TRELLO_TOKEN");
+        this.rt = Runtime.getRuntime();
+        Unirest.config().cookieSpec(CookieSpecs.IGNORE_COOKIES);
+    }
+
+    public String getBoardsFromTrello() {
+        String jsonString = "";
+        try {
+            String curl = "curl https://api.trello.com/1/members/me/boards?fields=name,url&key=" + trelloApiKey + "&token=" + trelloApiToken;
```

Comment:
Suggest rename this to `curlCommand`.

```suggestion
            String curlCommand = "curl https://api.trello.com/1/members/me/boards?fields=name,url&key=" + trelloApiKey + "&token=" + trelloApiToken;
```

And also suggest do not directly execute a system command for anything, this operation is danger, which may causes some security issues(such as `RCE remote command executions, etc.`), and may not accept in the real-world productions.

But for our develop speed, this may not very important, so you can stil ignore this.